documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -6,6 +6,37 @@
 
 * No prerequisites
 
+**Note:** This step is deprecated. Use the [isChangeInDevelopment](isChangeInDevelopment.md) instead.
+
+You can keep most of the step specific configuration parameters in your configuration file `config.yml` untouched. The new step support the old naming convention. However, it is recommended to adjust your parameters to the new steps.
+
+Following parameters are not supported anymore. Adjust as indicated.
+
+| Unsupported Parameter | Change Notice |
+| ------------- | ------------- |
+| changeManagement/`<type>`/docker/envVars | Use `dockerEnvVars` instead. |
+| changeManagement/`<type>`/docker/image | Use `dockerImage` instead. |
+| changeManagement/`<type>`/docker/options | Use `dockerOptions` instead. |
+| changeManagement/`<type>`/docker/pullImage | Use `dockerPullImage` instead. |
+| changeManagement/git/format | This parameter has been dropped. Make sure that your change document IDs and transport request IDs are part of the Git commit message body. |
```

Comment:
- This parameter has been deprecated* ?

- I don't understand the last sentence, is it "make sure that you change your document IDs and transport request IDs" or is it "Make sure that you change your document IDs and that your transport request IDs are part of the Git commit message body."?